core/tests/protractor_utils/editor.js

```diff
@@ -1065,7 +1065,9 @@ var discardChanges = function() {
   element(by.css('.protractor-test-save-discard-toggle')).click();
   element(by.css('.protractor-test-discard-changes')).click();
   general.acceptAlert();
+  var availableElement = by.css('oppia-loading-modal-body');
```

Comment:
I don't understand this line, since availableElement is not being used anywhere. What's it for? Can it be removed?